src/applications/disability-benefits/all-claims/tests/utils/utils.unit.spec.jsx

```diff
@@ -670,6 +743,47 @@ describe('isAnswering781aQuestions', () => {
     });
   });
 
+  describe('PTSD flow migration', () => {
+    const ptsdFormData = {
+      newDisabilities: [
+        {
+          condition: 'Ptsd personal trauma',
+        },
+      ],
+      'view:selectablePtsdTypes': {
+        'view:assaultPtsdType': true,
+      },
+      'view:upload781aChoice': 'answerQuestions',
+      secondaryIncident0: {
+        'view:uploadSources': true,
+      },
+    };
+
+    describe('when the modern 0781 flow is enabled', () => {
+      const ptsdFormDataModernFlowEnabled = {
+        syncModern0781Flow: true,
+        ...ptsdFormData,
+      };
+
+      it('should return false so the legacy 0781a pages are hidden', () => {
+        expect(isAnswering781aQuestions(0)(ptsdFormDataModernFlowEnabled)).to.be
+          .false;
+      });
+    });
+
+    describe('when the modern 0781 flow is disabled', () => {
+      const ptsdFormDataModernFlowDisabled = {
+        syncModern0781Flow: false,
+        ...ptsdFormData,
+      };
+
+      it('should return true so the legacy 0781a flow is visible', () => {
+        expect(isAnswering781aQuestions(0)(ptsdFormDataModernFlowDisabled)).to
```

Comment:
I think `.be` is missing at the end of this line.

Reply:
Good catch, I didn't know this but it looks like `.to` and `.be` are syntactical sugar in Chai and don't actually do anything (https://devhints.io/chai), so that's why this tests still works.

Anyway, I tried to change it back for consistency and now the linter won't let me?

<img width="374" alt="Screenshot 2025-02-24 at 4 36 30 PM" src="https://github.com/user-attachments/assets/d2f32bff-877e-453a-9b86-750d1c6e5f02" />